### PR TITLE
Modification of enum semantic attributes 

### PR DESCRIPTION
**Pull request by harshita19244**
Resolves https://github.com/open-telemetry/opentelemetry-java/issues/3006

### CODE REVIEW

File: buildscripts/semantic-convention/templates/SemanticAttributes.java.j2 [hunk 1]
```diff
@@ -75,7 +75,7 @@ public final class {{class}} {
   {%- for attribute in attributes | unique(attribute="fqn") %}
   {%- if attribute.is_enum %}
   {%- set class_name = attribute.fqn | to_camelcase(True) ~ "Values" %}
-  {%- set type = to_java_return_type(attribute.attr_type.enum_type) %}
+  {%- set type = to_java_return_type(attribute.attr_type.string_type) %}
   {%- if attribute.attr_type.custom_values %}
```
Comment (jkwatson) [inline buildscripts/semantic-convention/templates/SemanticAttributes.java.j2:79]: we should get rid of this `if` check and keep the type assignment as it was before

Comment (harshita19244) [reply buildscripts/semantic-convention/templates/SemanticAttributes.java.j2:79]: Right okay, I will make the changes

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -36,7 +36,12 @@ you must run a local docker daemon.
 check formatting:
 
     `./gradlew build`
+    
+4. If you are a Windows user, use the alternate command mentioned below to run tests and
```
Comment (jkwatson) [inline CONTRIBUTING.md:40]: I think these changes are already on the `main` branch, so you probably should rebase your branch against the upstream/main to have a clean PR

**Comment by jkwatson** [comment]
@anuraaga does this end up looking like you were thinking in #3006?

**Review by anuraaga — APPROVED** [review]
Yeah, thanks for the help!

**Comment by harshita19244** [comment]
Thanks @jkwatson @anuraaga

**Pull request merged**